src/apps/activities/domain/response_type_config.py

```diff
@@ -264,9 +264,8 @@ class PhrasalTemplateConfig(PublicModel):
     remove_back_button: bool
 
 
-class RequestHealthRecordDataConfig(PublicModel):
+class RequestHealthRecordDataConfig(_ScreenConfig):
     type: Literal[ResponseType.REQUEST_HEALTH_RECORD_DATA] | None
```

Comment:
This seems to be causing in a breaking change for activities that had an existing EHR item prior to this change. The new expectation of the non-nullable `skippable_item` boolean property causes those activities to fail validation because that property is absent for those items, making it impossible to even load them in the Admin App.

The below workaround seems to fix it, though I guess the "correct" approach would be to create a migration script. Hopefully @rcmerlo is fine with this approach?

```suggestion
    type: Literal[ResponseType.REQUEST_HEALTH_RECORD_DATA] | None
    # Add `| None` for backwards-compatibility with previously supported configuration
    skippable_item: bool | None
```

Reply:
Defaulting the value to `False` instead, to simplify the pre-existing EHR items update